index.html

```diff
@@ -1,13 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./main.css">
-    <title>Just templete</title>
+    <link rel="stylesheet" href="./index.css">
+    <title>Awesome Books</title>
 </head>
+
 <body>
-    <h1>Do you need a Quick templete for HTM, CSS, Sass and JavaScript</h1>
+    <nav>
+        <h2>Awesome Books</h2>
+        <ul>
+            <li><a id="list-link" href="#">List</a></li>
+            <li><a id="add-link" href="#">Add new</a></li>
+            <li><a id ="contact-link" href="#">Contact</a></li>
+        </ul>
+    </nav>
+    <div class="date">
+        <p>April 29th 2022</p>
+    </div>
```

Comment:
- [ ] Your date and time should be displayed using the `luxon` package. Kindly do this.